package.json

```diff
@@ -80,7 +80,6 @@
     "git-repo-version": "~1.0.1",
     "highlight.js": "~9.12.0",
     "html-loader": "~0.5.1",
-    "husky": "^0.14.3",
```

Comment:
Why did you remove this? Was it too annoying in your experience? We could change it so it runs on prepush.

Reply:
yep, especially during merging... I didn't want to commit the changes though. Prepush would be fine

Reply:
Im fine with this removal, too :D